cdap-data-fabric/src/main/java/co/cask/cdap/data2/datafabric/dataset/DatasetServiceClient.java

```diff
@@ -147,6 +148,17 @@ public DatasetMeta getInstance(String instanceName) throws DatasetManagementExce
     return GSON.fromJson(response.getResponseBodyAsString(), SUMMARY_LIST_TYPE);
   }
 
+  Collection<DatasetSpecificationSummary> getAllInstances(Map<String, String> properties)
```

Comment:
Should call if `getInstances` as it is not getting all instances.

Reply:
renamed to `getInstances`